style.css

```diff
@@ -1,468 +1,1187 @@
-*, *:before, *:after {
+*,
+*::before,
+*::after {
     padding: 0;
     margin: 0;
     border: 0;
     box-sizing: border-box;
 }
+
 body {
     margin: 0 auto;
     max-width: 1440px;
     font-family: 'Montserrat', sans-serif;
     font-style: normal;
-    background: #FFFFFF;
+    background: #fff;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Project_AnastassiaVarabei_Evgen/pullRequest?prid=6131199)